src/components/SelectBox.jsx

```diff
@@ -0,0 +1,51 @@
+import styles from '../styles/selectBox'
+
+import React from 'react'
+import ReactMarkdown from 'react-markdown'
+
+const LANGS_OPTIONS = ['en', 'fr']
+
+const Option = ({t, inputData, optionValue, setValue}) => (
+  <option
+    value={optionValue}
+    selected={setValue === optionValue}
+  >
+    {t(`AccountView.${inputData}.${optionValue}.text`)}
+  </option>
+)
+
+const SelectBox = ({t, inputData, setValue, infosSubmitting, updateInfos}) => (
+  <div className={styles['coz-form']}>
+    <h3>{t(`AccountView.${inputData}.title`)}</h3>
+    <label className={styles['coz-desc']}>
+      {t(`AccountView.${inputData}.label`)}
+    </label>
+    <select
+      name={inputData}
+      onBlur={updateInfos}
+      >
+      {
+        LANGS_OPTIONS.map(function (lang) {
+          return <Option
+            optionValue={lang}
+            inputData={inputData}
+            setValue={setValue}
+            t={t}
+          />
+        })
+      }
+    </select>
+    <p className={styles['coz-desc']}>
+      <ReactMarkdown
```

Comment:
@gregorylegarec @CPatchane Well I thought about that and I'm not sure splitting is a durable way. Sure, english and french have a similar grammar and sentence construction, but that's not always the case. Maybe, with other languages, the bold part won't be included at the same location (in the middle of the sentence), maybe it will have to be at the beginning or the end of the sentence. How are you gonna do it with templating and split strings? Change the position of the string on given locale ? and how translators are supposed to tell us when we need to add an exception ?
With ReactMarkdown, there's only one string to call for us and the translator, who knows better about grammar in foreign languages than us, decides where to put what (according to the existing construction we have in the english locale).
I'm not saying we should always use ReactMardown, just keep it as an option maybe, but splitting complex strings like this one won't work forever. I think, at least ReactMarkdown seems the most reliable & durable solution for that kind of case.
But I would love to get @m4dz & @goldoraf's opinions too.